src/sentry/api/endpoints/project_stats.py

```diff
@@ -60,6 +60,24 @@ def get(self, request, project):
             stat_model = tsdb.models.project
         elif stat == 'forwarded':
             stat_model = tsdb.models.project_total_forwarded
+        elif stat == 'ip_address':
+            stat_model = tsdb.models.project_total_received_ip_address
+        elif stat == 'release_version':
+            stat_model = tsdb.models.project_total_received_release_version
+        elif stat == 'error_message':
+            stat_model = tsdb.models.project_total_received_error_message
+        elif stat == 'browser_extensions':
+            stat_model = tsdb.models.project_total_received_browser_extensions
+        elif stat == 'legacy_browsers':
+            stat_model = tsdb.models.project_total_received_legacy_browsers
+        elif stat == 'localhost':
+            stat_model = tsdb.models.project_total_received_localhost
+        elif stat == 'web_crawlers':
+            stat_model = tsdb.models.project_total_received_web_crawlers
+        elif stat == 'invalid_csp':
+            stat_model = tsdb.models.project_total_received_invalid_csp
+        elif stat == 'cors':
+            stat_model = tsdb.models.project_total_received_cors
```

Comment:
I'd echo the comment @mattrobenolt made about a dictionary lookup here.

There's even a `BidirectionalMapping` if you want to share the same structure to track both sides: https://github.com/getsentry/sentry/blob/0598866e1c6ac9c67ebcccdc6a166b06dd126532/src/sentry/utils/datastructures.py#L8-L63